test/dynamo/test_repros.py

```diff
@@ -1674,6 +1674,77 @@ def fn(x, b, y):
         y = torch.randn(2, requires_grad=True)
         opt_fn(x, b, y)
 
+    def test_autograd_function_classmethod(self):
```

Comment:
Can you have a smaller repro for this?

Reply:
Can you move this to ```test/dynamo/test_autograd_function.py``` where should be the right place.